output-blocks/src/Control/OutputCapable/Blocks/Type.hs

```diff
@@ -86,6 +89,20 @@ getOutputSequenceWithRating
   -> m (Maybe Rational, [Output])
 getOutputSequenceWithRating = Generic.getOutputSequenceWithRating English
 
+{-|
+Converts 'OutputCapable' value using 'GenericOutput'
+into a result and a list of 'GenericOutput'
+
+Consider using 'getOutputSequenceWithRating'
+or even more specific versions of 'Control.OutputCapable.Blocks.Type'
+in order to get better error messages on implementation errors.
+-}
```

Comment:
This documentation should also be specialized? (It still talks about `GenericOutput`.)